src/ProgressOnderwijsUtils/Data/ParameterizedSqlObjectMapper.cs

```diff
@@ -472,12 +516,13 @@ public bool Equals(ColumnOrdering other)
                 static ByMetaObjectImpl()
                 {
                     var writablePropCount = 0;
+                    var debug = "";
                     foreach (var mp in metadata) { //perf:no LINQ
-                        if (mp.CanWrite && SupportsType(mp.DataType)) {
+                        if (mp.CanWrite && SupportsType(mp.DataType) || SupportsTypeByCustomCreateMethod(mp.DataType)) {

```

Comment:
haakjes verkeerd danwel ik zou een method maken voor het concept `SupportsType(mp.DataType) || SupportsTypeByCustomCreateMethod(mp.DataType)`

Reply:
volgens mij staat het haakje goed. Het is of een writable supported type, of een TypeByCustomCreate